fwoa-tools/package.json

```diff
@@ -27,7 +27,18 @@
     "sort-package-json": "sort-package-json package.json",
     "standard-version": "standard-version --skip.tag=true",
     "test": "heft test --clean --no-build && rushx pkg-json-lint",
-    "watch": "tsc -w"
+    "watch": "tsc -w",
+    "jest": "jest"
+  },
+  "jest": {
```

Comment:
Do we need this? Tests work locally without this block with `rushx test`

Reply:
This allows running individual unit tests in an IDE. Without this, the tests could not run in Webstorm. It gave this error. I believe this configuration is required to be able to run the test with the `jest` command instead of with `rush`
```
 ● Test suite failed to run

    SyntaxError: /Users/thingut/workplace/fhir-works-on-aws/fwoa-tools/src/migrationExport.test.ts: Missing initializer in const declaration. (30:22)

      28 |
      29 |             // eslint-disable-next-line @typescript-eslint/no-explicit-any
    > 30 |             const argv: any = parseCmdOptions();
         |                       ^
      31 |             expect(argv.dryRun).toEqual(true);
      32 |             expect(argv.smart).toEqual(true);
      33 |             expect(argv.since).toEqual("1800-01-01T00:00:00.000Z");

```